Project.toml

```
@@ -1,15 +1,17 @@
 name = "FixedPointDecimals"
 uuid = "fb4d412d-6eee-574d-9565-ede6634db7b0"
 authors = ["Fengyang Wang <fengyang.wang.0@gmail.com>", "Curtis Vogt <curtis.vogt@gmail.com>"]
-version = "0.5.4"
+version = "0.5.5"
 
 [deps]
 BitIntegers = "c3b6d118-76ef-56ca-8cc7-ebb389d030a1"
 Parsers = "69de0a69-1ddd-5017-9359-2bf0b02dc9f0"
 
 [compat]
-Parsers = "2.7"
 BitIntegers = "0.3.1"
+Parsers = "2.7"
+Printf = ">=1.6"
+Test = ">=1.6"
```

As written these compat entries promise to be compatible with all future versions of Printf and Test, including e.g. a v2 with many breaking changes

I think we probably only want to declare compatibility with those that are SemVer compatible with 1.6, which we can do via:

```suggestion
Printf = "1.6"
Test = "1.6"
```

https://pkgdocs.julialang.org/v1/compatibility/#Version-specifier-format